docker-part2/exe-8/dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM tomcat:latest
+
+WORKDIR /usr/local/tomcat
+
+RUN mv webapps webapps2 && \
+    mv webapps.dist/ webapps
+
+EXPOSE 8080
```

Comment:
no need to write it again, we get all the other commands from the "FROM" command